src/site/content/en/blog/performance_tool_score_diffs.md

```diff
@@ -0,0 +1,179 @@
+---
+
+# Why is my performance score different between different tools? What is my REAL score?
+
+<a href="https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics">Load is not a single moment in time</a>—it's an experience that no one metric can fully capture. There are multiple moments during the load experience that can affect whether a user perceives it as "fast", and if you just focus on one you might miss bad experiences that happen during the rest of the time. <b>This is why we generally use multiple metrics blended together to create our performance scores;</b> but different tools use different metrics and different weighted blends. This can be frustrating when you're looking for a source of truth about how fast your website is.
+
+This blog post serves to explain the reasons that there are differences in performance scores between three Google tools: <a href="https://www.thinkwithgoogle.com/intl/en-gb/feature/testmysite">Test My Site (TMS)</a>, <a href="https://developers.google.com/speed/pagespeed/insights/">PageSpeed Insights (PSI)</a>, and <a href="https://developers.google.com/web/tools/lighthouse/">Lighthouse</a>. This document does <b>not</b> detail other differences in the tools, such as server run locations, network throttling presets, score variance between runs, etc.
+
+## Terminology
+
+Various terms about speed/performance are referenced in our tools. We want to start off with some clarification of how these terms are used.
+
+- <b>Page speed</b> refers to the speed of an individual page on your site (URL level).
+- <b>Site speed</b> refers to the speed of all of the pages on your site (origin level).
+
+It is important to note that ‘performance’ or ‘speed’ is not to be defined by a single metric (i.e. First Contentful Paint (FCP), Time to Interactive (TTI), etc.). This is motivated by several considerations, including:
+- (as mentioned before) Page "load is not a single moment in time—it's an experience that no one metric can fully capture" [<a href="https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics">source</a>]
+- Similarly, no single metric in isolation is able to give a meaningful indication of the health of the web (critical for benchmarking).
+- Depending on which metric is chosen, a very different picture of page speed is presented (each requiring different optimizations to improve).
+
+## Performance Scores within Tools
+
+The performance score (top level gauge, 1-100 score) in PSI and Lighthouse is calculated from a weighted blend of multiple metrics to summarize a website's/web app's speed. This <a href="https://docs.google.com/spreadsheets/d/1sH_T4G_RZAg4CpcV1bT-tmUegBdBpCOOwsdzqtWnO4U/edit#gid=0">weighted blend</a> of metrics that together constitute the performance score in both PSI and LH (as of May 2019, <a href="https://github.com/GoogleChrome/lighthouse/releases">LH v5.0</a>) is as follows:
+- 3X - First Contentful Paint (FCP)
```

Comment:
Do you want to link these to the corresponding docs on web.dev and /web. I think web.dev has docs for everything except FID.

https://web.dev/lighthouse-performance